architecture/service_catalog/index.adoc

```diff
@@ -101,6 +101,34 @@ references to the binding secret from the deployment. Otherwise, the next
 rollout will fail.
 ====
 
+ifdef::openshift-enterprise,openshift-origin[]
+[[service-catalog-route-planning]]
+== Service catalog compatibility with services outside of the cluster
+
+[NOTE]
+====
+This section is only relevant to clusters using the `NetworkPolicy` networking
+SDN plug-in.
+====
+
+The difference in networking policies can impact your service broker's
+accessibility. You must ensure that `NetworkPolicy` objects exist in your
+service broker, so that users can have appropriate access to these resources in
+their project or namespace. These `NetworkPolicy` objects are triggered by using
+the service catalog through provisioning or binding.
+
+For example, if the service catalog need to create an object inside of a user's
+project, then `NetworkPolicy` rules needs to allow access to the desired
+project. Or, if the service catalog creates objects in a different name space in
```

Comment:
Or, if the service catalog creates objects in a different name space in
the cluster, then the target project may need `NetworkPolicy` rules to allow the source project to talk to it.